### DISCUSSION TITLE
How to get the resource detail when using ResourceTemplate

### DISCUSSION THREAD

**Discussion by FounderHy** [argoproj/argo-workflows#7626] [Q&A] [answered]
Now I use ResourceTemplate to create SparkApplication, I want to know how I can get the SparkApplication generated name from the created workflow.

**Comment by terrytangyuan** [comment, marked as answer]
You can use jsonPath or jqFilter to access the created custom resource objects. Examples:
* https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-orchestration.yaml
* https://github.com/argoproj/argo-workflows/blob/master/examples/k8s-jobs.yaml
